lib/turn.rb

```diff
@@ -0,0 +1,15 @@
+class Turn
+  attr_reader :guess, :card
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    card.answer == guess
+  end
+
+  def feedback
+    correct? ? "Correct!" : "Incorrect! The correct answer was '#{card.answer}'"
```

Comment:
Ternary operator is good for cases like this, but can become confusing -- just be sparing with using it!

Reply:
Ternary op is another of the modern JS bread and butter tools. I'd be curious to learn more about why twould be confusing?